package.json

```diff
@@ -1,43 +1,48 @@
 {
   "name": "svg-typewriter",
-  "version": "0.3.0",
-  "description": "A library for measuring, manipulating, and writing text on SVG.",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/palantir/svg-textwriter.git"
-  },
-  "author": "Andrzej Skrodzki @ Palantir Technologies",
-  "main": "svgtypewriter.js",
-  "typings": "svgtypewriter.d.ts",
+  "version": "1.0.0",
+  "description": "SVG Text Measurement Line Wrapper",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "buildPath": "build",
+  "testsGlob": "build/test/*Tests.js",
   "scripts": {
-    "test": "grunt test-travis"
-  },
-  "devDependencies": {
-    "grunt": "~0.4.2",
-    "grunt-blanket-mocha": "~0.5.0",
-    "grunt-bump": "~0.7.0",
-    "grunt-cli": "~0.1.13",
-    "grunt-contrib-clean": "~0.7.0",
-    "grunt-contrib-compress": "~0.14.0",
-    "grunt-contrib-concat": "~0.5.1",
-    "grunt-contrib-connect": "~0.11.2",
-    "grunt-contrib-jshint": "^0.12.0",
-    "grunt-contrib-uglify": "~0.11.0",
-    "grunt-contrib-watch": "~0.6.1",
-    "grunt-mocha-phantomjs": "~2.0.1",
-    "grunt-parallelize": "1.1.3",
-    "grunt-saucelabs": "8.6.2",
-    "grunt-sed": "~0.1.1",
-    "grunt-shell": "~1.1.2",
-    "grunt-ts": "~5.3.2",
-    "grunt-tslint": "~3.0.1",
-    "grunt-typedoc": "~0.2.1",
-    "grunt-umd": "^2.3.5",
-    "load-grunt-tasks": "~3.4.0",
-    "tslint": "~3.7.4",
-    "typescript": "~1.8.9"
+    "build": "npm-run-all build:clean build:compile build:namespace",
+    "build:clean": "rimraf ${npm_package_buildPath}",
+    "build:compile": "tsc --project .",
+    "build:namespace": "echo \"\\nexport as namespace SVGTypewriter;\" >> ${npm_package_types}",
```

Comment:
whoa what is export as namespace? Is this some crazy Typescript magic that converts a module into a namespace?!